packages/swap/src/__tests__/integration.test.ts

```diff
@@ -142,6 +142,7 @@ describe('python integration test', () => {
       id: "doesn't matter",
       intermediateAmount: '2000000000',
       outputAmount: '0', // this shouldn't be the result
+      quoteType: 'broker',
```

Comment:
do we need to strip this from the response of the `/quote` endpoint? not sure if we want to expose it there (yet)

Reply:
okay i think you already did that 🙂